optuna/distributions.py

```diff
@@ -48,6 +48,22 @@ def to_internal_repr(self, param_value_in_external_repr):
 
         return param_value_in_external_repr
 
+    @abc.abstractmethod
+    def contains(self, param_value_in_internal_repr):
```

Comment:
`contains` is currently just for internal use in Optuna. Let's make this private so far.

Reply:
Sounds good. I have renamed the method name at the commit https://github.com/pfnet/optuna/pull/383/commits/81808983bfe0048a66dd0eddbdb468687eb2c504.